test/over_react/component/shared_error_boundary_tests.dart

```diff
@@ -290,11 +290,13 @@ void sharedErrorBoundaryTests(BuilderOnlyUiFactory builder) {
     });
 
     group('and consecutive errors are thrown from the same component', () {
-      Future<Null> triggerErrorsViaButtonClickThatSignifyAnUnrecoverableComponent() async {
+      Future<Null> triggerErrorsViaButtonClickThatSignifyAnUnrecoverableComponent({throwExtraError = false}) async {
```

Comment:
#nit coul d use typing
```suggestion
      Future<Null> triggerErrorsViaButtonClickThatSignifyAnUnrecoverableComponent({bool throwExtraError = false}) async {
```